### PR TITLE
Add X-Request-ID to application logs

### PR DESCRIPTION
**Pull request by pawbu**
Looks like `json_logging.get_correlation_id()` function with fastapi as web framework needs the request object to correctly infer the X-Request-ID in application logs.

### CODE REVIEW

File: application.py [hunk 1]
```diff
@@ -193,12 +194,13 @@ async def unknown_exception_handler(request, exc):
...
     response = await call_next(request)
-    response.headers["X-Request-ID"] = json_logging.get_correlation_id()
+    response.headers["X-Request-ID"] = json_logging.get_correlation_id(request=request)
     return response
 
 
 @data_service_app.on_event("startup")
 def startup_event():
+    json_logging.CREATE_CORRELATION_ID_IF_NOT_EXISTS = True
     json_logging.CORRELATION_ID_GENERATOR = lambda: "data-service-" + str(uuid.uuid1())
```
Comment (vaskalan) [inline application.py:204]: Looks like a new correlation id is generated on startup. 
But we want new correlation id pr. session (if client sends no id), not pr. startup.
Did I miss something?

**Review by vaskalan — APPROVED** [review]

File: application.py (hunk 1 above)
Comment (pawbu) [reply application.py:204]: Correlation ID will be created only if there is a request. If the request does not contain it, json_logging will create one.

Comment (vaskalan) [reply application.py:204]: Ok!

**Review by DanielElisenberg — APPROVED** [review]
Looks good, and thaught me something about fast-api :100:

**Pull request merged**